docker-entrypoint.sh

```diff
@@ -4,7 +4,8 @@ echo "ServerName $CUPSD_SERVER_NAME" > /etc/cups/client.conf
 TIMEOUT=120
 CUPSPORT=631
 CUPSHOST=cupsd
-if [ "$CUPSD_UPDATE" = "true" ]
+
```

Comment:
Might need to pull develop into branch as I changed this recently so we dont want to change it back

Reply:
Unless my change was wrong, in which case happy to use this new one